assests/javaScript/home.js

```diff
@@ -0,0 +1,39 @@
+const movieList = document.getElementById("movie-list");
+let markUp = "";
+
+fetch(
+  `https://api.themoviedb.org/3/movie/top_rated?api_key=d2aa1d6a7ef9316cd7e2270dd937b843&language=en-US&page=1`
+)
+  .then((res) => res.json())
+  .then((data) => showMovies(data.results))
+  .catch((error) => console.log(`error: ${error}`));
```

Comment:
`console.log` is not a best way to handle errors. Please show some message to end user on UI in case of failure.

Reply:
So I replace it by **alert**

Reply:
So I replace it by **alert**